go-controller/pkg/ovn/multihoming_test.go

```diff
@@ -214,8 +214,15 @@ func (em *secondaryNetworkExpectationMachine) expectedLogicalSwitchesAndPorts(is
 			var otherConfig map[string]string
 			if hasSubnets {
 				otherConfig = map[string]string{
-					"exclude_ips": managementPortIP(subnet).String(),
-					"subnet":      subnet.String(),
+					"subnet": subnet.String(),
+				}
+				if !ocInfo.bnc.IsPrimaryNetwork() {
+					// FIXME: This is weird that for secondary networks that don't have
```

Comment:
@maiqueb I don't have time to dig into this right now but headsup for you.

Reply:
tks. created https://github.com/ovn-org/ovn-kubernetes/issues/4764